### PR TITLE
Xfce additions

### PR DESCRIPTION
**Pull request by ZachBacon**
**Summary**

Brings the inclusion of xfce4-clipman-plugin and xfce4-battery-plugin

**Test Plan**

installed, and added to the plugin lists for the panel to verify if it worked. Battery one reported low battery, but I'm not on mobile hardware like a laptop. Some external testing would be nice.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/x/xfce4-clipman-plugin/pspec_x86_64.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    </Source>
+    <Package>
+        <Name>xfce4-clipman-plugin</Name>
+        <Summary xml:lang="en">Clipman is a clipboard manager for Xfce.</Summary>
+        <Description xml:lang="en">Clipman is a clipboard manager for Xfce. It keeps the clipboard contents around while it is usually lost when you close an application. It is able to handle text and images, and has a feature to execute actions on specific text by matching them against regular expressions.
+</Description>
+        <PartOf>desktop.xfce</PartOf>
+        <Files>
+            <Path fileType="config">/etc/xdg/autostart/xfce4-clipman-plugin-autostart.desktop</Path>
+            <Path fileType="config">/etc/xdg/xfce4/panel/xfce4-clipman-actions.xml</Path>
```
Comment (ReillyBrogan) [inline packages/x/xfce4-clipman-plugin/pspec_x86_64.xml:24]: Use stateless paths

Comment (ZachBacon) [reply packages/x/xfce4-clipman-plugin/pspec_x86_64.xml:24]: RIght, I'll fix that in both packages.

**Comment by ZachBacon** [comment]
Issue resolved for clipman, should be safe to merge in

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**